eth/protocols/snap/sync.go

```diff
@@ -2061,14 +2061,14 @@ func (s *Syncer) onByteCodes(peer *Peer, id uint64, bytecodes [][]byte) error {
 
 	// Cross reference the requested bytecodes with the response to find gaps
 	// that the serving node is missing
-	hasher := sha3.NewLegacyKeccak256()
-
+	hasher := sha3.NewLegacyKeccak256().(crypto.KeccakState)
+	var hash = make([]byte, 32)
```

Comment:
`hash := make([]byte, 32)`, Go usually either uses `var` for the default value or `:=` for non defaults. Pls also add a newline after this.

Pls update the others too.

Reply:
Squashfixed